src/Tasks/Move.cs

```diff
@@ -149,7 +157,7 @@ public override bool Execute()
 
                 try
                 {
-                    if (MoveFileWithLogging(sourceFile, destinationFile))
+                    if (!FailIfNotIncremental && MoveFileWithLogging(sourceFile, destinationFile))
```

Comment:
No need for a failure message in the failure case?

Reply:
IMO `Move` is, like `Touch`, inherently broken in incremental scenarios. It can be used within a target, but that target needs to skip.